pkg/remoteresolution/resolver/bundle/resolver.go

```diff
@@ -0,0 +1,78 @@
+/*
+ Copyright 2024 The Tekton Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package bundle
+
+import (
+	"context"
+
+	"github.com/tektoncd/pipeline/pkg/apis/resolution/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/remoteresolution/resolver/framework"
+	"github.com/tektoncd/pipeline/pkg/resolution/common"
+	"github.com/tektoncd/pipeline/pkg/resolution/resolver/bundle"
```

Comment:
This dependencies allows us to re-use code, which is nice. 
Do we need to keep the old one around though? Only the new one is built and deployed with pipeline, and we don't support mix and matching pipeline versions with built-in resolvers versions (same for all built-in resolvers)

Reply:
It's mostly there for backwards compatibility. In case users are making use of the older ones. We can deprecate it with the framework I think. It gives enough time for users to port over.

Reply:
I think it's ok to give a bit of notice if you want, however that part of the code is not covered by the API policy, so maybe in the next one or two of releases we can clean this up to reduce the unused code we maintain.